CONTRIBUTING.md

```diff
@@ -68,13 +78,18 @@ branch.
 ### PR & Merge Procedure
 
 - Ensure the PR branch is rebased on `main`.
+- Ensure you provided unit tests and integration tests.
 - Run `make test-unit test-e2e` to ensure that all tests pass.
 - Merge the PR!
 
-### Release Procedure
+## Release procedure
+
+We follow semver:
```

Comment:
Would be wise for us to state that we have not followed this guideline in the past, and at what point we are committing to this (e.g., once v3.0.0 is released)

Reply:
You mean the `2.0` release? nah... it's still a full release for umeemania, and v3.0 contains new features and breaking changes.

Reply:
But the patch version update 2.0.1 does have new features, not just containing hot fixes, and we've not used the "beta1/rc" syntax at all. It's alright as long as the team is aligned 👍

Reply:
If you prefer, I can rephrase to:
 
> We follow semver (from v3.0.0):

Reply:
Just openede https://github.com/umee-network/umee/pull/917 to address 👍